File: app/src/androidTest/java/au/com/realestate/hometime/BaseTest.java
package au.com.realestate.hometime;

import au.com.realestate.hometime.service.TramsRepository;

public class BaseTest {

    protected TestRepository mTestRepo;
    protected TramsRepository mTramsRepo;

    public void init() {
        mTestRepo = TestRepository.getInstance();
        mTramsRepo = mTestRepo.getTramsRepository();
    }

    public void addDelay(long delay) throws Exception {
        Thread.sleep(delay);
    }

}

File: app/src/androidTest/java/au/com/realestate/hometime/BaseTest.kt
package au.com.realestate.hometime

import au.com.realestate.hometime.service.TramsRepository

abstract class BaseTest {

    protected lateinit var testRepo: TestRepository
    protected lateinit var tramsRepo: TramsRepository

    protected val UILOAD_DELAY = 2000L

    open fun setup() {
        testRepo = TestRepository.getInstance()
        tramsRepo = testRepo.tramsRepository
    }

    fun addDelay(millis:Long){
        Thread.sleep(millis)
    }
}
